docs/documentation/functions/index.md

```diff
@@ -97,3 +97,28 @@ describe 'ensure_resource' do
   end
 end
 {% endhighlight %}
+
+## Mocking a function for use in all tests
+    
+You may not need to test a function itself, but a calling class may require the function
+to return a value. You can mock the function once and make it available to all test.
+Modify *spec/spec_helper_local.rb* and use `Puppet::Parser::Functions.newfunction()`
```

Comment:
`rspec-puppet` really doesn't mention `spec_helper_local.rb` anywhere. That's really a PDK-thing and I'd avoid it here.